oar/core/advisory_mgr.py

```diff
@@ -34,7 +34,8 @@ def get_advisories(self):
                 errata_id=v,
                 impetus=k,
             )
-            ads.append(ad)
+            if ad.errata_state != "DROPPED_NO_SHIP":
```

Comment:
suggest to add `DROPPED_NO_SHIP ` to `oar/core/const.py`

Reply:
the state check should be happened before `Advisory` initialization

Reply:
Thanks, @rioliu-rh. Pushed.